pkg/asset/agent/common/utility.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import "github.com/google/uuid"
+
+// InfraEnvID holds an uniuqe identifier for infra env resource.
+var InfraEnvID string
+
+func init() {
+	// Generate a UUID during initialization
+	InfraEnvID = uuid.New().String()
```

Comment:
Could this be an Asset? Generating a new one each time the installer is run makes me nervous.

Reply:
Also, unless strictly required, I'd suggest to avoid as much as possible the `init()` method, as in general it will make harder writing the tests